### PR TITLE
Remove cache query for reports url

### PR DESCRIPTION
**Pull request by kevinhaube**
This PR rolls back the cache query from the reports url

Test Steps:
1. `yarn` and `yarn start`
2. Login
3. Ensure reports still load on `/daily-data` page

## Changes
- Changed the reports url

### Testing
- [X] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [X] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Fixes
- #issue - List GitHub issues this PR fixes

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

**Review by sean-usds — APPROVED** [review]
let's go! 🟢

**Review by sethdarragile6 — APPROVED** [review]

File: frontend-react/src/pages/TermsOfService.tsx [hunk 1]
```diff
@@ -3,475 +3,520 @@ import { Alert } from "@trussworks/react-uswds";
 export const TermsOfService = () => {
     return (
         <>
-            <section className="usa-site-alert usa-site-alert--info margin-top-neg-4 desktop:margin-top-neg-8 margin-bottom-3 tablet:margin-bottom-6" id="anchor-top">
+            <section
```
Comment (TomNUSDS) [inline frontend-react/src/pages/TermsOfService.tsx:6]: This file seems like it was a prettier change only?
This is going to make merging with other branches _really_ painful.

**Review by TomNUSDS — COMMENTED** [review]
Maybe you can remove files that are prettier-only changes to minimize its impact on other pending PRs?

**Comment by kevinhaube** [comment]
@TomNUSDS I removed the prettier files but added `yarn.lock` because of the build error. I merged with the most recent on `master`. This _shouldn't_ cause errors when building once merged (famous last words). 🤣

**Review by TomNUSDS — APPROVED** [review]
LGTM

**Pull request merged**